contracts/core/Modifier.sol

```diff
@@ -15,6 +21,12 @@ abstract contract Modifier is Module, SignatureChecker, IAvatar {
     /// @param sender The address of the sender.
     error NotAuthorized(address sender);
 
+    /// @param hash already executed.
```

Comment:
Perhaps these two errors should live in `ExecutionTracker.sol` ?

Reply:
Ok. Should we keep ExecutionTracker? What about the Error names?

Reply:
Yeah, I think it's nice to have it split out.
Error names look good.

Reply:
Moved the errors.

ExecutionTracker still missing explanatory comments on fields, errors, and on the external function